R/utils.R

```diff
@@ -480,3 +480,11 @@ infer_scenario_type <- function(baseline_scenario, shock_scenario) {
     )
   }
 }
+
+end_year_lookup <- function(scenario_type) {
+  return(2040)
```

Comment:
this function will always return 2040 irrespecive of scenario_type as a function ends at the first return statement

Reply:
done i finally understand return